class TestyError(Exception):
    "raised when an error occurs in testy"
    pass